### ISSUE TITLE
revokeInvitatiion not working

### ISSUE THREAD

**Issue by cquiroz** [gemini-hlsw/lucuma-odb#1094]
I tried to delete an existing invitation but I get an error

mutation:
```graphql
mutation revokeInvitation($id: UserInvitationId!) {
          revokeUserInvitation(input: { id: $id }) {
            invitation {
              id
            }
          }
        }
```

with variables
```json
{
    "id": "10d"
}
```

And produces an error:
```json
[
    {
        "message": "Invitation does not exist, is no longer pending, or was issued by someone else.",
        "extensions": {
            "odb_error": {
                "odb_error.tag": "invitation_error",
                "odb_error.detail": "Invitation does not exist, is no longer pending, or was issued by someone else.",
                "odb_error.data": {
                    "invitationId": "10d"
                }
            }
        }
    }
]
```

**Comment by tpolecat** [comment]
Can you look at this test https://github.com/gemini-hlsw/lucuma-odb/blob/main/modules/service/src/test/scala/lucuma/odb/graphql/mutation/revokeUserInvitation.scala#L53 and see if that's what you're doing? Normal science user creates and then revokes an invitation.

**Comment by cquiroz** [comment]
yes, that's pretty much what I'm doing though in not one go, I get the ids of the existing invites and then do a revoke

**Comment by tpolecat** [comment]
Resolved (see discussion in #1101)
